content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,182 @@
+:guilabel:`Client Secret Value` into a notepad as it will become encrypted after leaving this page.
+The :guilabel:`Secret ID` is not needed.
+
+.. image:: azure_oauth/secretvalue.png
+   :align: center
+   :alt: Client Secret Value or Value in the app's credentials.
+
+After these steps, the following items should be ready to be set up in Odoo:
+- A client ID (:guilabel:`Client ID` or :guilabel:`Application ID`)
+- A client secret (:guilabel:`Value` or :guilabel:`Client Secret Value`)
+
+This completes the setup on the :guilabel:`Microsoft Azure Portal` side.
+
+Setup in Odoo
+=============
+
+Enter Microsoft Outlook credentials
+-----------------------------------
+
+First, open the Odoo database and navigate to the :guilabel:`Apps` module. Then, remove the
+:guilabel:`Apps` filter from the search bar and type in `Outlook`. After that, install the module
+called :guilabel:`Microsoft Outlook`.
+
+Next, navigate to :guilabel:`Settings --> General Settings`, and under the :guilabel:`Discuss`
+section, ensure that the checkbox for :guilabel:`External Email Servers` is checked. This populates
+a new option for :guilabel:`Outlook Credentials`.
+
+:guilabel:`Save` the progress.
+
+Then, copy and paste the :guilabel:`Client ID` (Application ID) and :guilabel:`Client Secret
+(Client Secret Value)` into the respective fields and :guilabel:`Save` the settings.
+
+Configure outgoing email server
+-------------------------------
+
+On the :guilabel:`General Settings` page, under the :guilabel:`External Email Servers` setting,
+click the :guilabel:`Outgoing Email Servers` link to configure the Microsoft account.
+
+Then, create a new email server and check the box for :guilabel:`Outlook`. Next, fill in the
+:guilabel:`Description` (it can be anything) and the Microsoft Outlook email :guilabel:`Username`.
+
+Then, cick on :guilabel:`Connect your Outlook account`.
+
+A new window from Microsoft opens to complete the :guilabel:`authorization process`. Select the
+appropriate email address that is being configured in Odoo.
+
+.. image:: azure_oauth/verify-outlook.png
+   :align: center
+   :alt: Permission page to grant access between newly created app and Odoo.
+
+Then, allow Odoo to access the Microsoft account by clicking on :guilabel:`Yes`. After this, the
+page will navigate back to the newly configured :guilabel:`Outgoing Mail Server` in Odoo. The
+configuration automatically loads the :guilabel:`token` in Odoo, and a tag stating
+:guilabel:`Outlook Token Valid` appears in green.
+
+Finally, click :guilabel:`Test Connection`. A confirmation message should appear. The Odoo database
+can now send safe, secure emails through Microsoft Outlook using OAuth authentication.
+
+.. tip::
+   To extend the server's use, set a :guilabel:`mail.force.smtp.from` system parameter so that email
+   is sent from a default “from email address” for all emails sent from the database. The value of
```

Comment:
Would italicize this since we're indirectly referring to a field/gui element.

```suggestion
   is sent from a default *from email address* for all emails sent from the database. The value of
```